rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -27,10 +27,23 @@ Create and document the environment for the automation of processes.
 
 |===
 +
-Learn how to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[Assign Privileges to a User].
-* To use AWS credentials, you need an AWS account.
+Learn how to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign privileges to a user].
+* To create AWS credentials, you need an AWS account.
 +
-Learn about https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS Configuration basics^]
+Learn about https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS configuration basics^]
+* To create OAuth credentials, you need an Azure account to create an application for Outlook.
++
+Learn about https://azure.microsoft.com/en-us/[Azure^].
++
+Learn how to https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/how-to-authenticate-an-ews-application-by-using-oauth[authenticate an EWS application by using OAuth^]
++
+Learn how to https://learn.microsoft.com/en-us/exchange/client-developer/legacy-protocols/how-to-authenticate-an-imap-pop-smtp-application-by-using-oauth[authenticate an IMAP, POP or SMTP connection using OAuth^]
++
+If you want to use RPA Builder to generate a refresh token for your OAuth credentials, you must have RPA Builder access assigned.
++
+Learn how to xref:usermanagement-manage.adoc#assign-users-rpa-builder-access[assign users RPA Builder access].
++
+Learn how to xref:rpa-builder::toolbox-variable-handling-credentials-for-oauth.adoc#generating-an-oauth-refresh-token[generate an OAuth refresh token].
 * Global variables and credentials can be linked to activity parameters defined in automations. The activity parameters for a process are defined in RPA Builder. Linking global variables or credentials overwrites the initial value of the linked activity parameters in a run configuration.
```

Comment:
Can you change the passive text to active?

You can link global variables and credentials to...